helm_helpers/Chart.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: v2
+type: library
+name: helm_helpers
```

Comment:
Why we use new name here for the same purpose? Why not "helm_lib"? The directory "helm_lib" could be preserved.

Reply:
Yuriy and I decided to name the directory helm_helpers, as he becomes a chart and uses lib-helm as a dependency.

Reply:
Well, it's definitely [multiplying entities beyond necessity](https://en.wikipedia.org/wiki/Occam%27s_razor).

I think "helm_lib" chart in Deckhouse and "deckhouse_lib" chart as it's dependency should be distinguished. In Deckhouse code, helm lib is a just a lib, as always, regardless the dependencies it is using. Like "python_lib" is called "deckhouse" in pip, while it is still in the repo deckhouse/python_lib and in Deckhouse, we have python_lib. 

It's just different contexts. Why shouldn't they be accounted? 

So, I'd not agree on that.

Reply:
fixed. revert helm_lib directory